src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -622,6 +624,12 @@
             <argument type="service" id="sylius.sql_logger.purger" />
         </service>
 
+        <service id="sylius.shipping_methods_resolver.by_zones_and_channel" class="%sylius.shipping_methods_resolver.by_zones_and_channel.class%">
+            <argument type="service" id="sylius.repository.shipping_method" />
+            <argument type="service" id="sylius.zone_matcher" />
+            <tag name="sylius.shipping_method_resolver" type="by_zones_and_channel" label="By zones and channel" priority="1" />
```

Comment:
Label should be a translation key: ``sylius.shipping_methods_resolver.by_zones_and_channel``.